openfpga_flow/openfpga_shell_scripts/fix_heterogeneous_device_example_script.openfpga

```diff
@@ -43,7 +43,7 @@ build_architecture_bitstream --verbose --write_file fabric_independent_bitstream
 build_fabric_bitstream --verbose
 
 # Write fabric-dependent bitstream
-write_fabric_bitstream --file fabric_bitstream.bit --format plain_text
+write_fabric_bitstream --file fabric_bitstream.bit --format plain_txt
```

Comment:
Is this a typo? I double checked documentation: https://openfpga.readthedocs.io/en/master/manual/openfpga_shell/openfpga_commands/fpga_bitstream_commands/#cmdoption-format